OpenUpSA/wazimap-ng-ui
# Use xss filter to sanitize html text for points data

## Description

Used package [xss](https://www.npmjs.com/package/xss) to sanitize Text HTML from backend.

## Related Issue
https://wazimap.atlassian.net/jira/software/c/projects/WNCM/boards/2?modal=detail&selectedIssue=WNCM-417


## How is it tested automatically?
Added jest test to check multiple conditions for point data

## How should a reviewer test it locally
* Add html tags in location object
   There are 2 ways to do that : 
   1) Upload a new csv point file with html tags in description
   2) Change data field of location object in admin panel
        
<img width="1791" alt="Screenshot 2022-06-20 at 10 59 01 AM" src="https://user-images.githubusercontent.com/6871866/174531290-d3632202-850b-40fb-a5dc-39b35567cacc.png">

* Add config in Profile collection
   ```
   field_type : {
      "KEY1" : "html",
      "KEY2": "text"
   }
   ```
* On frontend check if html tags are rendered correctly in point data description modal

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
